apps/webapp/app/routes/_app.orgs.$organizationSlug.projects.$projectParam.env.$envParam.runs.$runParam/route.tsx

```
@@ -1448,6 +1547,17 @@ function KeyboardShortcuts({
   );
 }
 
+function AdjacentRunsShortcuts({
+}) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)